src/components/crown/crownButtons/crownColorDropdown.vue

```diff
@@ -23,7 +23,7 @@
 
       <button
         type="button"
-        class="color-button"
+        class="color-button clear-color fa fa-redo-alt"
```

Comment:
You could use the `p-0` bootstrap helper class here instead of `clear-color`, if it gives you a warm fuzzy feeling.

Reply:
+1 :)